lib/pbench/server/database/models/users.py

```diff
@@ -147,9 +147,10 @@ def is_admin(self) -> bool:
 
     @staticmethod
     def is_admin_username(username) -> bool:
-        # TODO: Need to add an interface to fetch admins list instead of
-        # hard-coding the names, preferably via sql query.
+        """Determine if the given user name is the offical admin user.
+
+        Returns:
+            True if the user name is "admin", False otherwise.
+        """
         admins = ["admin"]
         return username in admins
```

Comment:
I don't mind losing the `TODO`, but I think the docstring could use tweaking.  Because
- we won't necessarily have a "**the** official admin user", as indicated for the moment by the fact that the check looks to see if the specified username is in the _list_ of admin users (which, admittedly, _currently_ contains only one entry, but might contain more in the future...), and
- likewise, the comment at line 153 makes an assertion which line 156 (taken alone) doesn't justify

I think the text should run more like "Determine if the given username is _an_ admin user", and "Returns `True` if the user is an admin; `False` otherwise."

Reply:
Sure, I'll update.

Reply:
Fixed.